packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -25,8 +25,14 @@ Downloaded by pub (not CocoaPods).
   # Versions earlier than 8.4 can't be supported because that's the first version
   # that supports privacy manifests.
   s.dependency 'GoogleMaps', '>= 8.4', '< 10.0'
+  s.dependency 'Google-Maps-iOS-Utils', '>= 5.0.0', '< 7.0.0'
```

Comment:
Please put a comment about the mapping from `GoogleMaps` versions to these versions, to explain why there's a range.

Reply:
I don't think this is related to GoogleMaps versions, more to supported iOS versions, 14 and 15.
Should we remove this range and let cocoapods to automatically resolve proper supported version or should we keep is as the dependency is tested only with versions 5 and 6.

Reply:
> Should we remove this range

We should not remove the range; see lines 21 and 22 of this file.

> and let cocoapods to automatically resolve proper supported version

Cocoapods has no way of knowing whether the code in this plugin will work with `Google-Maps-iOS-Utils` version 7, so cannot resolve the proper supported version if there is no upper bound.